### DISCUSSION TITLE
ANN: Presentation mode for slideshows

### DISCUSSION THREAD

**Discussion by williamstein** [sagemathinc/cocalc#6475] [Announcements]
CoCalc has a very new graphical [Slides](https://github.com/sagemathinc/cocalc/discussions/6420) [editor](https://cocalc.com/features/slides), with rich text support, computational Jupyter code cells, pens so you can write during your presentation, and more.  This is  based on [our Whiteboard](https://cocalc.com/features/whiteboard) so it has many of the same features as the whiteboard (e.g., mind mapping edges, etc.)

I just added a **new slideshow presentation mode**, which makes things cleaner when you want to **actually give a presentation** using a slideshow.  To use it, click on the present icon:  
<img width="52" alt="image" src="https://user-images.githubusercontent.com/1276278/221013742-5667b7a9-4ed0-4972-93e8-4cdd78053f2b.png">
or select "Slideshow Presentation" from the dropdown:
<img width="227" alt="image" src="https://user-images.githubusercontent.com/1276278/221013805-f107ace7-8e6c-49e0-8828-af78241e9b1e.png">
then click the "Start Slideshow" button:

<img width="269" alt="image" src="https://user-images.githubusercontent.com/1276278/221013863-2e69ded6-0789-4290-bd3a-ca15ae74cc38.png">

This will transition to fullscreen with only the actual slide displayed (nothing outside the margins), always maximally zoomed, like in Powerpoint or Google slides.  There's a little wrench icon (that appears only when you move the mouse)

<img width="182" alt="image" src="https://user-images.githubusercontent.com/1276278/221014077-837e5670-e999-47f7-be7d-43756d08fb55.png">

you can click it to toggle the toolbar, in case you want to select a pen or click on code and run it live during your talk, or access any other editing tools. 

PLANS:

- there is still no presenter control panel, which links speaker notes to the fullscreen presentation window.
- there is no conversion between these slides and slides created using the Jupyter notebook slide toolbar. That's planned and will enable live execution of code during talks, along with the ability to type notes, scribble with a pen, etc.

**Comment by DrXyzzy** [comment]
Online docs updated: https://doc.cocalc.com/slides.html#presentation-mode

**Reply by williamstein** [reply]
I added links to this from the slides help button.
